code-challenges.js

```diff
@@ -3,8 +3,32 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+const fibonacci = () => {
+    //declare an array that will hold the numbers of the sequence, containing the first two already for something to work with.
+    let fibArr = [0,1]
+    let num1 = fibArr [0]
+    let num2 = fibArr [1]
+    let next = 
+    //we're starting the count at 2 because we already have the first 2 numbers.
+    count = 2;
+    //create a loop that will continue to run until we reach 10 iterations.
+    while (count <= 10){
+        //add the first two numbers of the sequence (index 0 and 1)
+        //the next number in the sequence is the sum of the first 2 numbers.
+        next = num1 + num2
+        //reassigning num1 to be the next number1 in the equation.
+        num1 = num2
+        //reassigning the new sum to be the next number2 in the equation.
+        num2 = next
+        //push the sum onto the array
+        fibArr.push(next)
+        //increment our number count
+        count++
+    }
+    return fibArr
+}
+
+// console.log(fibonacci());
```

Comment:
Excellent code. I want to mention that it's important to look at the expected outcome. Your solution is great, but unfortunately your output includes the 0, when it needs to start with 1. Not a biggy for this, but definitely something I'd like you to keep a lookout for.